lib/src/Uncertainty/Bayesian/Gibbs.cxx

```diff
@@ -96,28 +143,35 @@ Point Gibbs::getRealization() const
   // see which components will need to recompute the posterior
   if (samplesNumber_ == 0)
   {
-    Point lp(getDimension());
-    for (UnsignedInteger j = 0; j < samplers_.getSize(); ++ j)
+    Point lp(nbSamplers);
```

Comment:
lp is not very explicit. Does it refer to a notation in a paper, or is it an abbreviation?

Reply:
I guess Julien meant "log-posterior". I have replaced "lp" with "samplersLogPosteriors".